app/src/ereferrals/java/org/eyeseetea/malariacare/data/database/utils/PopulateDBStrategy.java

```diff
@@ -59,7 +62,12 @@ public InputStream openFile(Context context, String table) throws IOException,
 
     @Override
     public void logoutWipe() {
-
+        List<Survey> surveys = Survey.getAllSurveys();
+        for (Survey survey : surveys) {
+            survey.delete();
+        }
+        ICredentialsRepository credentialsLocalDataSource = new CredentialsLocalDataSource();
```

Comment:
@manuelplazaspalacio this class is only for wipe db, can you move clean up ereferrals credentials to AuthenticationLocalDataSourceStrategy as normal credentials for more readable code? And instead of save null credentials better if we create a clearCredentials method as in AuthenticationLocalDataSource for has more readable code.